react/wbooks/src/app/components/SignUp/index.js

```diff
@@ -48,13 +43,26 @@ class SignUp extends Component {
               inputType={get(FIELDS_DATA[field], 'inputType')}
             />
           ))}
-          <button type="submit" className={`${styles.signUpButton} full-width m-top-4`}>
+          <button
+            type="button"
```

Comment:
by removing the `type="submit"` you are doing a `GET` request in to your own code, also if you are doing this by this behaviour (which is weird) you have to add a `event` arg in the `handleSignUp` function and do the `event.preventDefault()`